src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const object = {
```

Comment:
```suggestion
  return {
```
And never use names like `object` :)

Just FYI